<launch>

  <!--arg name="arm" default="l"/-->
  <!--arg name="controller" default="static"/-->
  <arg name="ee_pos_threshold" default="0.05"/>
  <arg name="ee_quat_threshold" default="0.7"/>

  <param name="haptic_mpc/pr2/ee_motion_threshold" value="$(arg ee_pos_threshold)"/>
  <param name="haptic_mpc/pr2/ee_quat_motion_threshold" value="$(arg ee_quat_threshold)"/>

  <node name="arm_reacher" pkg="hrl_base_selection" type="arm_reacher_server.py" output="screen">
  </node>


</launch>
